pages/index.tsx

```diff
@@ -1,15 +1,23 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
-import "./index.css";
+//import "./index.css";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
 
-const root = ReactDOM.createRoot(document.getElementById("root"));
+/*const root = ReactDOM.createRoot(document.getElementById("root"));
```

Comment:
Clear commented code. Don't commit commented code is not worth it. For keeping track of old code you have git.

Reply:
done